python/vyos/ifconfig/interface.py

```diff
@@ -1227,7 +1266,12 @@ def update(self, config):
         if 'is_bridge_member' in config:
             bridge_dict = config.get('is_bridge_member')
             self.add_to_bridge(bridge_dict)
-
+        
+        # Re-set rules for the mirror monitoring interface
+        if 'is_monitor_intf' in config:
+            mirror_rules = config.get('is_monitor_intf')
+            self.apply_mirror_of_monitor(mirror_rules)
```

Comment:
When we have `self._config` avaliable as class member, why pass a sub key of the dict as argument?